test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -161,17 +161,21 @@ class CheckYourAnswersControllerSpec extends SpecBase with ControllerMockFixture
         }
 
         "must redirect to Information sent when UserAnswers is empty" in {
-          val application = applicationBuilder(userAnswers = Option(emptyUserAnswers), AffinityGroup.Individual)
-            .overrides(bind[Navigator].toInstance(new FakeNavigator(onwardRoute)))
-            .build()
+          forAll(indWithId.arbitrary) {
```

Comment:
When working on the test where `UserAnswers is empty`, then shouldn't use `forAll` as the `indWithId.arbitrary` is a specially designed to have all the valid answers, whereas if you just need the empty then no need to use any arbitrary.